commcare_connect/opportunity/api/serializers.py

```diff
@@ -105,6 +114,15 @@ def get_deliver_progress(self, obj):
         opp_access = _get_opp_access(self.context.get("request").user, obj)
         return opp_access.visit_count
 
+    def get_max_visits_per_user(self, obj):
+        return obj.paymentunit_set.aggregate(max_total=Sum("max_total")).get("max_total", 0)
+
+    def get_daily_max_visits_per_user(self, obj):
+        return obj.paymentunit_set.aggregate(max_daily=Sum("max_daily")).get("max_daily", 0)
+
+    def get_budget_per_visit(self, obj):
+        return obj.paymentunit_set.aggregate(amount=Sum("amount")).get("amount", 0)
```

Comment:
This seems like it will sum over all visits (since each unit is a different visit) not per visit. I think ideally this would be a weighted average (max_total*amount for each type / overall_total), but if thats too complicated this can be `max(amount)` for now (if there are two visit types worth 1 and 2 dollars, still better to say 2 than 3).

Reply:
Cool, I will make it max for now.